package.json

```diff
@@ -44,7 +44,7 @@
     "react-native-background-timer": "2.1.1",
     "react-native-barcode-builder": "^1.0.5",
     "react-native-calendar-events": "1.7.1",
-    "react-native-camera": "1.2.0",
+    "react-native-camera": "2.1.1",
```

Comment:
Why not [2.11.1](https://www.npmjs.com/package/react-native-camera/v/2.11.1)?

Reply:
I forgot a 1, I'll update immediately.